### PR TITLE
Omar/carousel

### PR DESCRIPTION
**Pull request by OmarG247**
- Added carousel for vehicles 
- Added more interfaces 
- Some styling changes

### CODE REVIEW

File: v2/src/components/VehicleCard.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+const totalDelay = (minutes: number) => {
+  const hours = Math.floor(minutes / 60);
+  let finalTime = '';
+
+  if (hours === 1) {
+    finalTime += `${hours} hour`;
+  } else if (hours > 1) {
+    finalTime += `${hours} hours`;
+  }
```
Comment (Arryan) [inline v2/src/components/VehicleCard.tsx:12]: shorter way of doing it maybe?
```javascript
finalTime += `${hours} hour${hours > 1 ? 's' : ''}`
```

File: v2/src/components/VehicleCard.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <div className={`transit-icon-container ${props.type}`}>
+        <img className="transit-icon" src={props.icon} alt="current vehicle icon" />
+      </div>
+      <h3 className="vehicle-card-header">
+        <b>Route {props.route}</b>
+      </h3>
+    </div>
+    <hr className="side-menu-divider" />
+    <p>
+      {props.direction} -> <b>{props.terminal}</b>
```
Comment (Arryan) [inline v2/src/components/VehicleCard.tsx:35]: suggestion: http://xahlee.info/comp/unicode_arrows.html

Comment (OmarG247) [reply v2/src/components/VehicleCard.tsx:35]: Good idea, I made the ones we have currently but if its more efficient let's use this. Thoughts?

File: v2/src/components/VehicleCard.tsx (hunk 1 above)
Comment (OmarG247) [reply v2/src/components/VehicleCard.tsx:12]: added more logic to both

Comment (OmarG247) [reply v2/src/components/VehicleCard.tsx]:
```
  return (hours > 0 ? (`${hours} hour${hours > 1 ? 's' : ''}`) : '') 
    + (hours === 0 ? `${minutes} minutes` : `, ${minutes % 60} minutes`);
```

**Pull request merged**